protobuf/types/execution.proto

```diff
@@ -119,4 +122,9 @@ message Execution {
   int64 blockHeight = 15 [
     (gogoproto.moretags) = 'hash:"-"'
   ];
+
+  // confs is the number of conrimations the exeuction needs to have before chaingin status to passed
+  int64 confs = 16 [
+    (gogoproto.moretags) = 'hash:"-"'
```

Comment:
If we add this data in the execution, then I think the confirmations should be part of the hash, 2 executions should be different if they have 2 different consensuses.